test-data/unit/fine-grained-modules.test

```diff
@@ -1632,3 +1632,112 @@ class Foo:
 [out]
 ==
 a.py:3: error: Argument 1 to "foo" of "Foo" has incompatible type "int"; expected "str"
+
+[case testSkipButDontIgnore1]
+# cmd: mypy a.py c.py
+# flags: --follow-imports=skip
+[file a.py]
+import b
+from c import x
+[file b.py]
```

Comment:
Maybe include something here that would trigger an error to verify that this actually gets skipped? Maybe also update `b.py`?